pulp_container/app/token_verification.py

```diff
@@ -61,7 +68,26 @@ def authenticate(self, request):
         if request.headers.get("Authorization") == "Basic Og==":
             return (AnonymousUser, None)
 
-        return super().authenticate(request)
+        try:
+            return super().authenticate(request)
+        except AuthenticationFailed:
+            if any(
+                self.PULP_AUTHENTICATION_CLASS in auth_class for auth_class in self.AUTH_CLASSES
```

Comment:
is not self.AUTH_CLASSES a tuple? you could just do if `self.PULP_AUTHENTICATION_CLASS n self.AUTH_CLASSES` or i am missing something?

Reply:
Yes, I should be able to do what you have sketched.